Assets/Scripts3D/Weapons/Weapon.cs

```diff
@@ -1,24 +1,104 @@
+using Assets.Helpers;
 using UnityEngine;
 
 public abstract class Weapon : MonoBehaviour
 {
-    [SerializeField]
-    protected Camera Camera;
-
     [SerializeField]
     protected Transform AttackPoint;
+    [SerializeField] public Camera Camera;
```

Comment:
```suggestion
    [SerializeField] 
    public Camera Camera;
```
For this and all the below